x/evm/keeper/statedb.go

```diff
@@ -82,7 +82,7 @@ func (k *Keeper) SetBalance(ctx sdk.Context, addr common.Address, amount *big.In
 	case 1:
 		// mint
 		coins := sdk.NewCoins(sdk.NewCoin(params.EvmDenom, sdk.NewIntFromBigInt(delta)))
-		if err := k.bankKeeper.MintCoins(ctx, types.ModuleName, coins); err != nil {
+		if err := k.potKeeper.SafeMintCoins(ctx, types.ModuleName, coins); err != nil {
```

Comment:
@alexstratos we have params.EvmDenom (constant) and types.ModuleName (from db), maybe let's switch to a constant?